src/Collector/CollectorTrait.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace Yiisoft\Yii\Debug\Collector;
+
+trait CollectorTrait
+{
+    private bool $isActive = false;
+
+    public function startup(): void
+    {
+        $this->isActive = true;
+    }
+
+    public function shutdown(): void
+    {
+        $this->isActive = false;
+    }
+
+    public function isActive(): bool
```

Comment:
Should it be part of the interface?

Reply:
Maybe, but why?

Reply:
I've thought it's used somewhere outside of collector implementation. It's not so no need for an interface. Making it private is a good idea though.